app/server.py

```diff
@@ -277,6 +277,26 @@ def restock_action(prod_id):
     Restock new quantity.
 
     This endpoint will update the number of new_qty of the given prod_id.
+    ---
+    tags:
+        - Inventory
+    parameters:
+        - name: prod_id
+          in: path
+          description: ID of product.
+          type: integer
+          required: true
+        - name: restock_amt
+          in: path
```

Comment:
I thought this parameter should be included in payload (i.e. in: data) instead of in path right?

Reply:
![image](https://user-images.githubusercontent.com/25423569/39450055-f69c6ad4-4c97-11e8-8d1e-586b9b3246cc.png)

Reply:
You are right.
Changed from "path" to "data".